[2025-04-26T22:28:00.164+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-26T22:28:00.226+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_csv_pipeline.load_data manual__2025-04-26T22:27:43.581660+00:00 [queued]>
[2025-04-26T22:28:00.256+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_csv_pipeline.load_data manual__2025-04-26T22:27:43.581660+00:00 [queued]>
[2025-04-26T22:28:00.268+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-26T22:28:00.311+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): load_data> on 2025-04-26 22:27:43.581660+00:00
[2025-04-26T22:28:00.331+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=104) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-26T22:28:00.334+0000] {standard_task_runner.py:72} INFO - Started process 114 to run task
[2025-04-26T22:28:00.349+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_csv_pipeline', 'load_data', 'manual__2025-04-26T22:27:43.581660+00:00', '--job-id', '79', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmpz40h0w09']
[2025-04-26T22:28:00.352+0000] {standard_task_runner.py:105} INFO - Job 79: Subtask load_data
[2025-04-26T22:28:00.507+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_csv_pipeline.load_data manual__2025-04-26T22:27:43.581660+00:00 [running]> on host f4244ab5d28e
[2025-04-26T22:28:00.840+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_csv_pipeline' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-26T22:27:43.581660+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-26T22:27:43.581660+00:00'
[2025-04-26T22:28:00.872+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-04-26T22:28:00.877+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-04-26T22:28:00.878+0000] {logging_mixin.py:190} INFO - Current task name:load_data state:running start_date:2025-04-26 22:28:00.228154+00:00
[2025-04-26T22:28:00.879+0000] {logging_mixin.py:190} INFO - Dag name:etl_csv_pipeline and current dag run status:running
[2025-04-26T22:28:00.888+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-26T22:28:04.055+0000] {load.py:34} INFO - âœ… Datos guardados en: data/output/superstore_clean.csv
[2025-04-26T22:28:04.057+0000] {load.py:35} INFO - ðŸ“Š Resumen: 9994 filas y 20 columnas
[2025-04-26T22:28:04.069+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-26T22:28:04.173+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-26T22:28:04.195+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=etl_csv_pipeline, task_id=load_data, run_id=manual__2025-04-26T22:27:43.581660+00:00, execution_date=20250426T222743, start_date=20250426T222800, end_date=20250426T222804
[2025-04-26T22:28:04.348+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-04-26T22:28:04.356+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-04-26T22:28:04.357+0000] {logging_mixin.py:190} INFO - Dag name:etl_csv_pipeline queued_at:2025-04-26 22:27:43.619794+00:00
[2025-04-26T22:28:04.358+0000] {logging_mixin.py:190} INFO - Task hostname:f4244ab5d28e operator:PythonOperator
[2025-04-26T22:28:04.457+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-26T22:28:04.518+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-26T22:28:04.627+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
